30-remote-site-setup.sh

```diff
@@ -1,60 +1,52 @@
 #!/bin/bash
 
-. /usr/local/bin/foreach_bosco_endpoint.sh
-
 BOSCO_KEY=/etc/osg/bosco.key
+REMOTE_HOST_KEY=`ssh-keyscan -H "$REMOTE_HOST"`
```

Comment:
Where does `REMOTE_HOST` getting set?

Reply:
From the environment, see slateci/slate-catalog#219

Reply:
@edquist ping

Reply:
Alright.  It wasn't super clear in the scripts here what was expected to be set in the script vs from outside (perhaps a comment somewhere about the expected externally-set environment might be useful), but in any case sounds like it should work.